apple/Sources/FerrostarSwiftUI/Views/CurrentRoadNameView.swift

```diff
@@ -0,0 +1,31 @@
+import SwiftUI
+
+public struct CurrentRoadNameView: View {
+    let currentRoadName: String?
+    let theme: any RoadNameViewTheme
+
+    public init(currentRoadName: String?,
+                theme: any RoadNameViewTheme = DefaultRoadNameViewTheme())
+    {
+        self.currentRoadName = currentRoadName
+        self.theme = theme
+    }
+
+    public var body: some View {
+        if let currentRoadName {
+            Text(currentRoadName)
+                .font(theme.textFont)
+                .foregroundStyle(theme.textColor)
+                .padding(.leading, 12)
+                .padding(.trailing, 12)
+                .padding(.vertical, 8)
+                .background(theme.backgroundColor)
+                .clipShape(.rect(cornerRadius: 48))
+                .overlay(RoundedRectangle(cornerRadius: 48).stroke(theme.borderColor, lineWidth: 1))
```

Comment:
Would be nice to have the & clip shape in the init or theme (similar to android comment). To allow easy customization.